src/app/window-handler.ts

```diff
@@ -196,6 +196,7 @@ export class WindowHandler {
         },
         {
           preload: path.join(__dirname, '../renderer/_preload-main.js'),
+          enableRemoteModule: true,
```

Comment:
I think we can move this into `getWindowOpts` instead of updating it in all the places